tests/tests/test-rpc/test-starknet-rpc.ts

```diff
@@ -541,4 +541,39 @@ describeDevMadara("Starknet RPC", (context) => {
       expect(error.message).to.equal("40: Contract error");
     }
   });
+
+  it("pendingTransactions", async function () {
+    await rpcTransfer(
+      providerRPC,
+      ARGENT_CONTRACT_NONCE,
+      ARGENT_CONTRACT_ADDRESS,
+      MINT_AMOUNT
+    );
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const txs: { type?: any; sender_address?: string; calldata?: string[] }[] =
```

Comment:
why not putting a type `string` or `enum` ? is the type returned from starknet-js wrong?